### ISSUE TITLE
u-boot for A5X Max - RK3228?

### ISSUE THREAD

**Issue by KarsteinE** [hexdump0815/u-boot-misc#1]
Can I use one of the ready builds for my A5X Max? If so which? And how to flash it?

**Comment by hexdump0815** [comment]
all the following is without any guarantee that it will work and without any support - if it works its good if it does not work, then it does not work: you may try to write this u-boot-image-for-emmc-h96max-rk3328.img: https://github.com/hexdump0815/linux-mainline-and-mali-rockchip-rk33xx-kernel/raw/master/misc.rkc/u-boot/u-boot-image-for-emmc-h96max-rk3328.img this way: https://github.com/hexdump0815/u-boot-misc/blob/master/readme.rk3328-no-sd-boot#L31-L41 to your emmc and then use this kernel: https://github.com/hexdump0815/linux-mainline-and-mali-rockchip-rk33xx-kernel/releases/tag/5.6.13-stb-rkc%2B and the included rk3328-h96max.dtb ... as said: if it works its good, if not it does not work ...

**Comment by KarsteinE** [comment]
Thanks, I'll give it a try during the weekend :)

**Comment by KarsteinE** [comment]
Nope, no luck. I only get Read LBA Failed! and Write LBA Failed! whatever I try....
